DEVELOPERS.md

```diff
@@ -83,6 +83,14 @@ A `.clang-format` configuration is provided with our defaults, please run source
 files through `clang-format` (or a formatter that produces equivalent behavior)
 before finalizing PR for submission.
 
+The `.clang-format` is the google style (as of feb 27 2024), except with 
+`SortIncludes` set to `false` to avoid breaking copybara path substitutions 
```

Comment:
Sorting is generally desirable. An alternative is to `// clang-format off` only where it would break us, what do you think?

Reply:
i believe that would almost work but i think there would still be corner cases that can slip through, eg a PR introduces a new #include. I can't think of anything that's entirely leak proof atm. For now, I'll use your barrier suggestion re-enable sorting.

Reply:
Right, I like the idea of a next_line/include/end atomic unit. One caveat there is that any // comment after include would break the build, which is a concern.